### PR TITLE
Feature/4619 get and put canned data

### PR DESCRIPTION
**Pull request by RuthBerry96**
Story: [AB#4619](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/4619)
Task: [AB#7196](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/7196) & [AB#7197](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/7197)

### CODE REVIEW

File: src/NHSD.BuyingCatalogue.Ordering.Api/Controllers/CommencementDateController.cs [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        public ActionResult Get()
+        {
+            var result = new CommencementDateModel { CommencementDate = DateTime.UtcNow };
+
+            return Ok(result);
+        }
+
+        [HttpPut]
+        public ActionResult Update([FromRoute] [Required] string orderId,
+            [FromBody] [Required] CommencementDateModel model)
```
Comment (bjss-chris) [inline src/NHSD.BuyingCatalogue.Ordering.Api/Controllers/CommencementDateController.cs:23]: The parameter attributes aren't needed and can be safely removed.

File: src/NHSD.BuyingCatalogue.Ordering.Api/Controllers/SupplierController.cs [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+                    EmailAddress = "bob.smith@email.com",
+                    TelephoneNumber = "4342345223  3434324"
+                }
+            };
+
+            return Ok(result);
+        }
+
+        [HttpPut]
+        public ActionResult Update([FromRoute] [Required] string orderId, [FromBody] [Required] SupplierModel model)
```
Comment (bjss-chris) [inline src/NHSD.BuyingCatalogue.Ordering.Api/Controllers/SupplierController.cs:45]: The parameter attributes aren't needed and can be safely removed.

File: src/NHSD.BuyingCatalogue.Ordering.Api/Controllers/CommencementDateController.cs (hunk 1 above)
Comment (RuthBerry96) [reply src/NHSD.BuyingCatalogue.Ordering.Api/Controllers/CommencementDateController.cs:23]: 👍

File: src/NHSD.BuyingCatalogue.Ordering.Api/Controllers/SupplierController.cs (hunk 2 above)
Comment (RuthBerry96) [reply src/NHSD.BuyingCatalogue.Ordering.Api/Controllers/SupplierController.cs:45]: 👍

**Pull request merged**